osu.Game/FodyWeavers.xml

```diff
@@ -0,0 +1,3 @@
+﻿<Weavers xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="FodyWeavers.xsd">
+  <Realm />
+</Weavers>
```

Comment:
One of these is already present in the solution/repository root; was this meant to be committed?

Reply:
No, was an accident. Happens when I do a build on the isolated project (which had local references to `osu.Game`) for whatever reason.